src/components/communities/CodeOfConductPage.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from "react";
+import { StyleSheet } from "react-native";
+
+import {
+  getCommunity,
+  loadCommunity,
+  useCommunity,
+} from "../../contexts/communityContext";
+import { useToken } from "../../contexts/tokenContext";
+import {
+  CodeOfConductPageNavigationProp,
+  CodeOfConductPageRouteProp,
+} from "../../routes";
+import Box from "../themed/Box";
+import Text from "../themed/Text";
+
+// React Navigation Types and Page Options
+
+type CodeOfConductPageProps = {
+  navigation: CodeOfConductPageNavigationProp;
+  route: CodeOfConductPageRouteProp;
+};
+
+export const CodeOfConductPageOptions = {
+  title: "Code Of Conduct",
+};
+
+// Page Styles
+
+const styles = StyleSheet.create({
+  root: {
+    flex: 1,
+    flexDirection: "column",
+    justifyContent: "flex-start",
+  },
+});
+
+// Page Definition
+
+export default function BannedUsersPage({ route }: CodeOfConductPageProps) {
+  // Load Community Dependencies
+  const [communityState, communityDispatch] = useCommunity();
+  const [tokenState, tokenDispatch] = useToken();
+
+  // Utilize community data
+  const [codeOfConduct, setCodeOfConduct] = useState("");
+
+  useEffect(() => {
```

Comment:
I think this would be a good use for `useConstructor`. no need to fix it just wanted to point it out

Reply:
Gotcha, I will likely be refactoring code this weekend so I will make note of that and utilize that

Reply:
Awesome